### DISCUSSION TITLE
How do I set up the password protection?

### DISCUSSION THREAD

**Discussion by 803516** [UseInterstellar/Interstellar#729] [Q&A] [answered]
How do I set up the password protection? Under the config file? can you please have a screenshot of what I do to do this. Thanks

![Screenshot 2024-05-30 10 28 02 AM](https://github.com/UseInterstellar/Interstellar/assets/169082567/199f8bd8-924f-4991-954f-a14bf4b52b2e)

**Comment by 0mhx** [comment]
It's pretty self explanatory

**Comment by nvd123YT** [comment]
set challenge to true

**Reply by 803516** [reply]
what do I do after I set it to true?

**Reply by nvd123YT** [reply, marked as answer]
> what do I do after I set it to true?

Where it says interstellar: password

Interstellar is the username
password is the password

To set password protection either leave it or change those to whatever

**Reply by 803516** [reply]
Ok thanks.

**Comment by 803516** [comment]
I have a new issue.

@803516 ➜ /workspaces/Interstellar (main) $ pnpm i && pnpm start
Lockfile is up to date, resolution step is skipped
Already up to date
Done in 559ms

> interstellar@5.2.3 start /workspaces/Interstellar
> node index.js

Password protection is enabled. Usernames are: interstellar
Passwords are: password
node:events:497
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::8080
    at Server.setupListenHandle [as _listen2] (node:net:1898:16)
    at listenInCluster (node:net:1946:12)
    at Server.listen (node:net:2044:7)
    at file:///workspaces/Interstellar/index.js:118:8
    at ModuleJob.run (node:internal/modules/esm/module_job:222:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:123:5)
Emitted 'error' event on Server instance at:
    at emitErrorNT (node:net:1925:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  code: 'EADDRINUSE',
  errno: -98,
  syscall: 'listen',
  address: '::',
  port: 8080
}

**Node.js v20.13.1
 ELIFECYCLE  Command failed with exit code 1.**

**Comment by 0mhx** [comment]
Check if you're running interstellar somewhere else

try closing all terminals on the codespace

**Comment by lakshinhemachandran** [comment]
enter this code exactly

challenge=true pnpm start

**Comment by lakshinhemachandran** [comment]
I think that is what it is

**Comment by lakshinhemachandran** [comment]
check the README file, it says everything
